src/components/Forms/EditPostForm/style.js

```diff
@@ -0,0 +1,13 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  editBtn: {
+    color: '#fff',
+    backgroundColor: '#3f51b5',
+    '&:hover': {
+      backgroundColor: '#7986cb',
+    },
+  },
+}));
```

Comment:
The button style looks different in chrome and safari.
<img width="914" alt="image" src="https://user-images.githubusercontent.com/42016928/69563128-803d0500-0f7e-11ea-9b4b-bb3a2286cd63.png">
<img width="919" alt="image" src="https://user-images.githubusercontent.com/42016928/69563155-8b903080-0f7e-11ea-8e18-f46a47043622.png">